deployments/ansible/molecule/custom_vars/converge.yml

```
@@ -12,7 +12,6 @@
     splunk_service_user: custom-user
     splunk_service_group: custom-group
     splunk_memory_total_mib: 256
```

Test the new option on linux:
```suggestion
    splunk_memory_total_mib: 256
    go_mem_limit: 230
```

---

done